### PR TITLE
add support for compressed secp256k1 keys encoded with base58btc

### LINKED ISSUE #41: Implement support for compressed public keys
Based off of https://github.com/status-im/status-go/issues/1937 the handler needs to be able to accept compressed public key.

@Samyoul which compressed formats do we need to support? Just multiformat base58 encoded?

### PR DESCRIPTION
**Pull request by jakubgs**
This adds support for Multibase compressed and uncompressed `secp256k1` public keys.

Examples of new paths:
* Lavender Trivial Goral
  * http://join.status.im/u/zQ3shuoHL7WZEfKdexM6EyDRDhXBgcKz5SVw79stVMpmeyUvG
  * http://join.status.im/u/f03e139115a1acc72510388fcf7e1cf492784c9a839888b25271465f4f1baa38c2d
  * http://join.status.im/u/fe70103e139115a1acc72510388fcf7e1cf492784c9a839888b25271465f4f1baa38c2d
* Enormous Vain Angwantibo
  * http://join.status.im/u/zQ3shPKfkQPeJdWMrXMtSfCFHhDR9gck99GsYt4Er3yvHxT2Z
  * http://join.status.im/u/f021c678bdeb6940df3e436cee76612ae6177f46e54e541fc024b572fafd9be89ba
  * http://join.status.im/u/fe701021c678bdeb6940df3e436cee76612ae6177f46e54e541fc024b572fafd9be89ba

Resolves #41.

### CODE REVIEW

File: utils/index.js [hunk 1]
```diff
@@ -42,11 +45,29 @@ const normalizeEns = (name) => (
 
 const showSpecialChars = (str) => univeil(str)
 
+/* check for multifactor variant encoding of the
+ * multicodec secp256k1 key identifier e7 */
+const isMultiFactorSecp256k1 = (bytes) => (
+  Buffer.from([231, 1]).compare(bytes.slice(0, 2)) == 0
+)
```
Comment (Samyoul) [inline utils/index.js:52]:
```suggestion
/* check for multiformat variant encoding of the
 * multicodec secp256k1 key identifier e7 */
const isMultiFormatSecp256k1 = (bytes) => (
  Buffer.from([231, 1]).compare(bytes.slice(0, 2)) == 0
)
```

File: utils/index.js [hunk 2]
```diff
@@ -42,11 +45,29 @@ const normalizeEns = (name) => (
...
+
+/* decodes base58btc encoding and decompresses a serialized secp256k1 */
+const decompressKey = (key) => {
+  let cBytes = multibase.decode(key)
+  if (isMultiFactorSecp256k1(cBytes)) {
```
Comment (Samyoul) [inline utils/index.js:57]:
```suggestion
  if (isMultiFormatSecp256k1(cBytes)) {
```

File: utils/index.js (hunk 1 above)
Comment (jakubgs) [reply utils/index.js]: done

File: utils/index.js (hunk 2 above)
Comment (jakubgs) [reply utils/index.js]: done

**Comment by jakubgs** [comment]
Deployed.

**Pull request merged**